LICENSE.md

```diff
@@ -1,20 +1,20 @@
 # Copyright {#about_copyright}
 
-| Summary                   |                                                                                        |
-|---------------------------|----------------------------------------------------------------------------------------|
-| Copyright holders         | Knut Reinert, [FU-Berlin](https://fu-berlin.de), [MPI MolGen](https://molgen.mpg.de/)  |
-| Source code               | ![](doc/about/copyright/bsdl_badge.svg)                                                |
-| Documentation             | ![](doc/about/copyright/ccby_badge.svg)                                                |
-| Snippets and examples     | ![](doc/about/copyright/cc0_badge.svg)                                                 |
-| Dependencies              | ![](doc/about/copyright/copyfree_badge.svg)                                            |
+| Summary                   |                                                                                            |
+|---------------------------|--------------------------------------------------------------------------------------------|
+| Copyright holders         | Knut Reinert, [FU-Berlin](https://fu-berlin.de), [MPI MolGen](https://www.molgen.mpg.de/)  |
+| Source code               | ![](about/copyright/bsdl_badge.svg)                                                        |
+| Documentation             | ![](about/copyright/ccby_badge.svg)                                                        |
+| Snippets and examples     | ![](about/copyright/cc0_badge.svg)                                                         |
+| Dependencies              | ![](about/copyright/copyfree_badge.svg)                                                    |
```

Comment:
The path changed from `doc/about` to `about/`. I don't know if the license will be rendered correctly on github?

Currently it renders correctly: https://github.com/seqan/learning-resources/blob/master/LICENSE.md